README.adoc

```diff
@@ -18,6 +18,22 @@
 //
 # MicroProfile OpenTracing
 
+'''
+== How to contribute
```

Comment:
Keep the formatting aligned with this file - so use `##`

Move the section before build

Reply:
Done